packages/web/src/components/common/reward-tooltip-content/RewardTooltipContent.tsx

```diff
@@ -123,10 +129,36 @@ const RewardTooltipContent: React.FC<RewardTooltipContentProps> = ({ rewardInfo
     });
   }, [externalRewards]);
 
+  const noneRewardUSD = useMemo(() => {
+    const isEmpty = !noneRewards || noneRewards?.length === 0;
+
+    if (isEmpty) return "-";
+
+    const sumUSD = noneRewards.reduce((accum: null | number, current) => {
+      if (accum === null && current.usd === null) {
+        return null;
+      }
+
+      if (accum === null) {
+        return current.usd;
+      }
+
+      if (current.usd === null) {
+        return accum;
+      }
```

Comment:
This condition is already filtered in the first conditional statement. It seems to be dead code.